client/src/global/components/form/Save.js

```diff
@@ -9,17 +9,21 @@ export default class FormSave extends Component {
   static propTypes = {
     text: PropTypes.string,
     cancelRoute: PropTypes.string,
-    theme: PropTypes.oneOf(["frontend", "backend", "reader"])
+    theme: PropTypes.oneOf(["frontend", "backend", "reader"]),
+    wide: PropTypes.bool
```

Comment:
Please remove this wide prop and the associated changes. I don't think the current select + button lockup is working. I realize that once this is removed the button will appear under the input, but that's fine because I'm going to refine the select + button interface in a different direction.